src/components/schemas/expense.yaml

```diff
@@ -769,4 +769,112 @@ expense_creation_from_receipt_in:
         This field takes the base64 encoded representation of the file.
       example: JVBERi0xLjQKMSAwIG9iago8PAovVGl0bGUgKP7/KQovQ3JlYX...
     source:
-      $ref: './fields.yaml#/source'
\ No newline at end of file
+      $ref: './fields.yaml#/source'
+
+spender_expense_check_policies_in:
+  type: object
+  additionalProperties: false
+  required:
+    - claim_amount
+    - spent_at
+  properties:
```

Comment:
Can you override description for all those fields which are not easily understandable/not looks proper?